client/components/footer.tsx

```diff
@@ -1,30 +1,30 @@
-import Container from './container'
-import { EXAMPLE_PATH } from '../lib/constants'
+import Link from 'next/link'
 
 const Footer = () => {
   return (
     <footer className="bg-neutral-50 border-t border-neutral-200">
-      <Container>
-        <div className="py-28 flex flex-col lg:flex-row items-center">
-          <h3 className="text-4xl lg:text-[2.5rem] font-bold tracking-tighter leading-tight text-center lg:text-left mb-10 lg:mb-0 lg:pr-4 lg:w-1/2">
-            Statically Generated with Next.js.
-          </h3>
-          <div className="flex flex-col lg:flex-row justify-center items-center lg:pl-4 lg:w-1/2">
-            <a
-              href="https://nextjs.org/docs/basic-features/pages"
-              className="mx-3 bg-black hover:bg-white hover:text-black border border-black text-white font-bold py-3 px-12 lg:px-8 duration-200 transition-colors mb-6 lg:mb-0"
-            >
-              Read Documentation
-            </a>
-            <a
-              href={`https://github.com/vercel/next.js/tree/canary/examples/${EXAMPLE_PATH}`}
-              className="mx-3 font-bold hover:underline"
-            >
-              View on GitHub
-            </a>
+    <div className="w-full mx-auto container md:p-8 p-6 md:flex md:items-center md:justify-between">
```

Comment:
cloud you use `@apply` from tailwind?

Reply:
create a "template" on tailwind config?

Reply:
Create one css per component

Reply:
you mean i create a css header with all the styles os header and it's childs? and so on

Reply:
ok i created one folder components/ inside of styles and more two css files into it.